common/src/main/java/software/tnb/common/config/OpenshiftConfiguration.java

```diff
@@ -76,6 +77,14 @@ public static String openshiftDeploymentLabel() {
         return getProperty(OPENSHIFT_DEPLOYMENT_LABEL, "app");
     }
 
+    /**
+     * set <a href="https://eclipse.dev/jkube/docs/openshift-maven-plugin/#jkube-openshift-deploymentconfig">jkube.build.switchToDeployment</a>
+     * @return boolean, true if jkube creates a Deployment, false to create a DeploymentConfig, default true
+     */
+    public static boolean forceOpenshiftDeployment() {
```

Comment:
shouldn't this be in SpringbootConfiguration, since it's for sb only? there are already `OPENSHIFT_SB_BASE_IMAGE` and `OPENSHIFT_SB_RESULT_IMAGE_REPOSITORY` anyway